app/controllers/events.js

```diff
@@ -0,0 +1,65 @@
+const bodybuilder = require('bodybuilder');
+const { get, isEmpty } = require('lodash');
+const uuid = require('uuid');
+
+const client = require('../services/elasticsearch');
+
+const searchEvents = (req, res, next) => client.search({
```

Comment:
may be getEvents is more convivient here? Because search - it is smth with filters or conditions.

Reply:
elasticSearch service use [.search](https://github.com/inst-event-emitter/eventemitter-backend/pull/10/files#diff-657f052bb6b4a7456e3173690b732c1dR7) method to find events by some criteria